File: src/LoveBook.Api/Dtos/CategoriesDto/GetAllCategories/GetCategoryByIdDto.cs
namespace Lovebook.Api.Dtos.CategoriesDto.GetAllCategories;

//public record GetCategoryByIdDto(long Id,string Name,HashSet<SpecDto> Specs);

public class GetCategoryByIdDto
{
    public long Id { get; set; }
    public string Name { get; set; }
    public bool IsActive { get; set; }
    public List<SpecDto> Specs { get; set; }
}

File: UI/src/libs/openapi-client/models/GetCategoryByIdDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { SpecDto } from './SpecDto';

export type GetCategoryByIdDto = {
    id?: number;
    name?: string | null;
    isActive?: boolean;
    specs?: Array<SpecDto> | null;
};
